src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompilationInfo.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.RegularExpressions
+{
+    public class RegexCompilationInfo
```

Comment:
As noted offline

- please expose this API for uap* also, if it's not going to work in netcoreapp either.
- please expose CompileToAssembly (which takes it) and make it throw PNSE.

Those are both to enable lightup /bailout.

Reply:
You can then verify that the baselines in src\shims are empty of all regex api 😸